### DISCUSSION TITLE
Using Storybook with Nunjucks (.njk) template

### DISCUSSION THREAD

**Discussion by talsingh1** [storybookjs/storybook#12014] [Help]
Hi,

I am trying to use the storybook-html with my nunjucks template. 
I am getting the below error which trying to run storybook - 

_Module parse failed: Unexpected token (1:0)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file._

I've also tried to configure nunjucks-loader in my webpack but i am still getting the same error.

```
import StartPage from "./header.njk";
var html = StartPage.render();
```

Thanks

**Comment by robpataki** [comment]
Hello @talsingh1, have you had any luck getting nunjucks templates render in Storybook?

**Reply by talsingh1** [reply]
Hi @robertpataki , we decided to go without storkbook in our project. Also, we faced similar issues while trying to render nunjucks macro's in jest unit test cases as well.

**Comment by GreatState** [comment]
Wer're also getting lots of errors as detailed here https://github.com/storybookjs/storybook/discussions/16658
